Library/Homebrew/requirements/apr_requirement.rb

```diff
@@ -4,11 +4,16 @@ class AprRequirement < Requirement
   fatal true
   default_formula "apr-util"
 
-  # APR shipped in Tiger is too old, but Leopard+ is usable
-  satisfy(build_env: false) { MacOS.version > :leopard && MacOS::CLT.installed? }
+  # APR shipped in Tiger is too old, but Leopard+ is usable.
+  # The *-config scripts were removed in Sierra, which is widely breaking.
+  satisfy(build_env: false) do
+    MacOS.version > :leopard && MacOS.version < :sierra &&
```

Comment:
Sorry, it would be `next if MacOS.version < :leopard` and `next if MacOS.version > :el_capitan`